project.mml

```diff
@@ -1453,6 +1453,23 @@ Layer:
         ) AS amenity_points_poly
     properties:
       minzoom: 10
+  - id: amenity-line
+    geometry: linestring
+    <<: *extents
+    Datasource:
+      <<: *osm2pgsql
+      table: |-
+        (SELECT
+          way,
+          COALESCE(
+           'highway_' || CASE WHEN tags->'ford' IN ('yes', 'stepping_stones') THEN 'ford' ELSE NULL END
```

Comment:
What about notation in this line - maybe it should be:

```SQL
'highway_' || WHEN tags @> 'ford=>yes' OR tags @> 'ford=>stepping_stones' THEN 'ford' ELSE NULL END
```

It's all too database related, so I'd like to have some kind of a rule of thumb when to use this notation and when it should be tags->'xxx'.

Reply:
Within the output expressions it's purely a style issue. Our guidelines currently call for `@>`. The style disadvantage to `@>` is that you have to repeat `tags @>` for each option instead of something like && for arrays

Reply:
I understand that this is not an error, because it was working also with the first notation and I tested that it still works. I can follow this style just to have clean and consistent code, I just don't know when should we use it and when not.

So what about this line?

Reply:
`@>` to be consistent with our style guide.

Reply:
Done.